Cartesian_kernel/include/CGAL/Cartesian/Point_2.h

```diff
@@ -47,6 +47,11 @@ class PointC2
   PointC2(const FT &x, const FT &y)
     : base(x, y) {}
```

Comment:
Is this constructor still useful? (same remark in almost every place a forwarding constructor is added)

Reply:
No, probably not. I am currently reviewing the code of this PR, and make attempts to simplify. But sometimes I get funny errors, like gmpxx complaining that it cannot be constructed from a `Point_2`.

That is a mess.